frappe/commands.py

```diff
@@ -533,6 +533,28 @@ def disable_scheduler(context):
 		finally:
 			frappe.destroy()
 
+
+@click.command('ready-for-migration')
+@click.option('--site', help='site name')
+@pass_context
+def ready_for_migration(context, site=None):
+	"Check if site is ready for migration"
+	if not site:
+		try:
+			site = context.sites[0]
+		except (IndexError, TypeError):
+			print 'Please specify --site sitename'
+			return 1
+	from frappe.utils.doctor import get_pending_jobs as _get_pending_jobs
+	frappe.init(site or '')
```

Comment:
If you init, you should destroy using try finally block. See other functions in commands.py.

Also, this command will always require site! so don't pass empty.